app/assets/stylesheets/response_view.sass

```diff
@@ -0,0 +1,40 @@
+.response
+  .estimate-and-variance
+    text-align: right
+    font-size: 0.8rem
+    font-color: $message-text-color
+  .creation-date
+    border: 2px solid $response-creation-date-border-color
+    background-color: $response-background-color
+    padding: 0.5rem 1.6rem
+    border-radius: $response-border-radius
+    font-size: 0.9rem
+    margin-top: 1rem
+    width: 100%
+    i
```

Comment:
Liever `.fa-clock-o` als selector, dat maakt de stylesheet in één opslag duidelijker dan `i`